src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Protocol/CustomNL2FxParams.cs

```diff
@@ -21,6 +21,11 @@ public class CustomNL2FxParams : BaseNLParams, IHasTextDocument
         /// Sentence in Natural Language.
         /// </summary>
         public string Sentence { get; set; }
+
+        /// <summary>
+        /// Name of the NL2Fx feature being used.
+        /// </summary>
+        public string FeatureName { get; set; }
```

Comment:
>FeatureName [](http://example.com/codeflow?start=22&length=11)

What are examples here? 

This payload is already specific to the NL2Fx feature (CustomNL2FxParams )